dlt-database/entity/0001-init_db/TransactionRecipe.ts

```diff
@@ -10,8 +10,8 @@ import {
 import { Decimal } from 'decimal.js-light'
 
 import { DecimalTransformer } from '../../src/typeorm/DecimalTransformer'
-import { Account } from '../Account'
-import { Community } from '../Community'
+import { Account } from './Account'
+import { Community } from './Community'
```

Comment:
the same as in the entity `Account`

Reply:
please add a comment above the code, why you make this change

Reply:
done.